src/main/webapp/app/exercises/programming/assess/repo-export/programming-assessment-repo-export-dialog.component.html

```diff
@@ -75,7 +75,13 @@ <h4 class="modal-title" jhiTranslate="entity.exportRepos.title">Confirm export o
         </div>
         <div *ngIf="!singleParticipantMode" class="checkbox">
             <label class="control-label">
-                <input class="form-check-input" type="checkbox" name="addParticipantName" [(ngModel)]="this.repositoryExportOptions.addParticipantName" />
+                <input
+                    class="form-check-input"
+                    type="checkbox"
+                    name="addParticipantName"
+                    [(ngModel)]="this.repositoryExportOptions.addParticipantName"
+                    [disabled]="!!participationIdList?.length"
```

Comment:
I have forgotten to also disable this checkbox if `!!participationIdList?.length` applies to stick to the logic before.